react/features/e2ee/middleware.js

```diff
@@ -116,6 +136,15 @@ MiddlewareRegistry.register(({ dispatch, getState }) => next => action => {
             dispatch(setEveryoneSupportE2EE(false));
         }
 
+        if (participantsCount >= MAX_MODE_LIMIT) {
```

Comment:
Let's think about extracting the calculation based on the participant count to a helper function.